apps/studio/src/lib/db/clients/bedrock.ts

```diff
@@ -0,0 +1,351 @@
+// BedrockClient - SQLite over MySQL protocol
+import { MysqlClient } from "./mysql";
+import { SqliteChangeBuilder } from "@shared/lib/sql/change_builder/SqliteChangeBuilder";
+import { SqliteData } from "@shared/lib/dialects/sqlite";
+import { ChangeBuilderBase } from "@shared/lib/sql/change_builder/ChangeBuilderBase";
+import { IDbConnectionServer } from "../backendTypes";
+import { IDbConnectionDatabase } from "../types";
+import {
+  ExtendedTableColumn,
+  TableTrigger,
+  TableIndex,
+  Routine,
+  FilterOptions,
+  TableOrView,
+  TableColumn,
+  SchemaFilterOptions,
+  DatabaseFilterOptions,
+  PrimaryKeyColumn,
+  TableProperties,
+  TableChanges,
+} from "../models";
+import {
+  IndexColumn,
+  TableKey as SharedTableKey,
+} from "@shared/lib/dialects/models";
+import _ from "lodash";
+import { parseVersion } from "@/common/version";
+import { buildSelectTopQuery } from "./utils";
+import logger from "@/lib/log/mainLogger";
```

Comment:
We don't import this directly, instead you should do this
```suggestion
import logger from "@bksLogger";

const log = logger.scope('bedrock.ts');
```
That will resolve to the proper logger for the file.
And creating the log object just makes it simpler to use, eg
```ts
log.info('Hello')
```